src/components/LoanCards/BorrowerInfo.vue

```diff
@@ -29,17 +30,39 @@ export default {
 		amount: {
 			type: String,
 			default: ''
+		},
+		status: {
+			type: String,
+			default: ''
+		},
+		borrowerCount: {
+			type: Number,
+			default: 1
 		}
 	},
 	computed: {
+		helpedLanguage() {
+			if (this.status === 'fundRaising' || this.status === 'inactive' || this.status === 'reviewed') {
+				return 'helps';
+			} return 'helped';
+		},
+		borrowerCountLanguage() {
+			if (this.borrowerCount > 1) {
+				return ' a member ';
+			} return ' ';
+		},
 		shortenedLoanUse() {
 			const maxLength = 100;
+			const nameLength = this.name.length;
+			const lowerCaseUse = this.use.toString().charAt(0).toLowerCase() + this.use.toString().slice(1);
+			const convertedUse = (this.use.substring(0, nameLength) === this.name) ? this.use : lowerCaseUse;
+
 			if (this.use.length === 0) {
 				return 'For the borrower\'s privacy, this loan has been made anonymous.';
-			} else if (this.use.length > maxLength) {
-				return `${this.use.substring(0, maxLength)}...`;
 			} else if (this.use.length < maxLength) {
```

Comment:
Currently if a loan use is exactly 100 chars it won't match either of your blocks so nothing will be returned.

You could make this line us <= so that a use of 100 characters would match...or

Another option would be move the > maxLength check block up to where then <  maxLength check is . Then just have an else (no if check needed) block that returns the convertedUse last in the chain...